arviz/tests/helpers.py

```diff
@@ -39,7 +39,6 @@ def chains():
 
 def create_model(seed=10):

```

Comment:
this function needs to remove the seed

Reply:
I would leave these seed here, because it is used to create different models. If we decide to remove the seed in `create_model`, we would need to modify also `create_multidimensional_model` **and** all the calls to these 2 functions.

Reply:
So should just leave the seed here or make the required changes to `create_multidimensional_model`?